### PR TITLE
added cricbuzz icon

### LINKED ISSUE #10295: Request: Cricbuzz
### Brand Name

Cricbuzz

### Website

https://www.cricbuzz.com/

### Popularity Metric

The similarweb rank is **166**. See [https://www.similarweb.com/website/cricbuzz.com](https://www.similarweb.com/website/cricbuzz.com/)

### Official Resources for Icon and Color

Example: svg [see svg logo](https://static.cricbuzz.com/images/cb_logo.svg)

### Additional Comments

All of their products found on their website [cricbuzz.com](https://www.cricbuzz.com/) all use the favicon as the main logo.
There are two variants of the logo : 
- One can be seen on the navbar of there website.
- another can be seen as favicon [see here](https://www.cricbuzz.com/images/apple-touch-icon.png)

### PR DESCRIPTION
**Pull request by praveen-rikhari**
**Issue:** closes #10295 

**Similarweb rank:** [166](https://www.similarweb.com/website/cricbuzz.com/)

### Checklist

- [x] I updated the JSON data in `_data/simple-icons.json`
- [x] I optimized the icon with SVGO or SVGOMG
- [x] The SVG `viewbox` is `0 0 24 24`

### Description

  - Why did you pick the hex value?
    HEX value is copied from their website [see here](https://www.cricbuzz.com/images/apple-touch-icon.png)
  - Did you manually vectorize the logo?
    Yes, the logo was manually vectorized from their web [source](https://www.cricbuzz.com/images/apple-touch-icon.png)

### CODE REVIEW

**Review by PeterShaggyNoble — CHANGES_REQUESTED** [review]
Thanks for the contribution, @praveen-rikhari 👍🏻 But the linter is currently failing as the precision in your icon exceeds our allowed maximum of 5 decimal places.

**Comment by praveen-rikhari** [comment]
@PeterShaggyNoble Thanks for the guidance and I have fixed the precision values

File: _data/simple-icons.json [hunk 1]
```diff
@@ -3390,6 +3390,11 @@
             "hex": "000000",
             "source": "https://www.crew-united.com"
         },
+        {
+            "title": "Cricbuzz",
+            "hex": "009270",
+            "source": "https://www.cricbuzz.com/images/apple-touch-icon.png"
```
Comment (PeterShaggyNoble) [inline _data/simple-icons.json:3396]: Also, you can just use the website for the `source`.
```suggestion
            "source": "https://www.cricbuzz.com"
```

**Review by PeterShaggyNoble — CHANGES_REQUESTED** [review]
@praveen-rikhari, there's an SVG available in the header of the website you could use here; I'm seeing [some significant differences](https://github.com/simple-icons/simple-icons/assets/15157491/cda31611-b733-49b6-bb24-a25137916717) between it and your version (in red).

**Comment by praveen-rikhari** [comment]
@PeterShaggyNoble I didn't get it is the svg wrong in any way?? Also my version in red i didn't get that part, can you clarify please

**Comment by PeterShaggyNoble** [comment]
That image shows the cricket ball from the source SVG in the website header in black with the SVG from this PR overlaid in red to show the differences between the 2. We need to match the source SVG as close as we can.

File: icons/cricbuzz.svg
Comment (adamrusted) [inline icons/cricbuzz.svg:1]: Seeing some differences between your SVG in black and the source SVG in red.
image

I've included the path I've extracted from source below. It'll need running through SVGO before you can submit it - but it's a good starting point!

```
M24,11.988C24,22.669 11.087,28.016 3.535,20.465C-4.016,12.913 1.331,0 12.012,0C18.633,0 24,5.367 24,11.988ZM1.36,7.157C1.174,7.557 1.016,7.97 0.887,8.392L1.169,9.117C1.295,8.71 1.443,8.31 1.612,7.918L1.36,7.157ZM12.761,0.629L12.641,0.923C12.965,0.869 13.301,0.797 13.636,0.797L13.714,0.599L13.672,0.599C13.361,0.599 13.061,0.629 12.761,0.629ZM0.833,12.222C0.791,12.642 0.773,13.067 0.779,13.499L0.779,13.816L1.067,14.236C1.073,13.786 1.109,13.337 1.157,12.899L0.833,12.222ZM3.62,6.54L3.566,5.503C3.209,5.954 2.883,6.428 2.589,6.923L2.805,7.834C3.057,7.391 3.326,6.959 3.62,6.54ZM5.79,3.285L5.718,4.178C6.1,3.846 6.5,3.536 6.917,3.249L7.169,2.356C6.69,2.637 6.23,2.947 5.79,3.285ZM11.299,13.786L8.679,13.786C8.491,14.711 7.643,15.363 6.701,15.309C5.215,15.309 4.693,14.494 4.957,12.618C5.221,10.741 5.97,9.926 7.456,9.926C8.499,9.926 8.967,10.388 9.015,11.449L11.634,11.449C11.706,9.045 10.537,8.02 7.726,8.02C4.393,8.02 2.781,9.399 2.331,12.618C1.882,15.836 3.105,17.221 6.425,17.221C9.243,17.221 10.699,16.184 11.299,13.786ZM17.496,17.221L17.484,17.239C20.182,17.233 21.53,15.854 21.98,12.636C22.43,9.417 21.476,8.038 18.767,8.038C17.786,7.98 16.814,8.262 16.016,8.835L16.615,4.675L13.996,4.675L12.258,17.041L14.308,17.041L14.781,16.208C15.321,16.897 16.208,17.221 17.496,17.221ZM19.391,12.618L19.391,12.636C19.127,14.512 18.426,15.327 17.065,15.327C15.704,15.327 15.231,14.482 15.494,12.636C15.758,10.789 16.459,9.926 17.82,9.926C19.181,9.926 19.654,10.741 19.391,12.618ZM15.848,0.785C15.315,0.688 14.777,0.624 14.236,0.593L14.176,0.743L14.835,0.743C15.177,0.743 15.512,0.785 15.848,0.785ZM12.204,0.294L12.138,0.45C12.431,0.45 12.737,0.42 13.025,0.42C13.313,0.42 13.624,0.42 13.942,0.456C13.368,0.356 12.786,0.302 12.204,0.294ZM0.659,14.218L0.521,13.96C0.604,14.445 0.72,14.924 0.869,15.393C0.784,14.997 0.72,14.597 0.677,14.194L0.659,14.218ZM1.073,14.985L0.821,14.638C0.9,15.569 1.078,16.49 1.354,17.383C1.19,16.594 1.095,15.791 1.073,14.985ZM7.75,1.079C7.302,1.251 6.866,1.451 6.443,1.678L6.329,2.398C6.701,2.159 7.085,1.941 7.48,1.744L7.75,1.079ZM9.368,0.599C8.987,0.683 8.611,0.79 8.242,0.917L8.002,1.517C8.385,1.349 8.781,1.199 9.201,1.067L9.368,0.599ZM10.675,0.36C10.375,0.396 10.076,0.438 9.788,0.492L9.614,0.893C9.92,0.809 10.214,0.731 10.537,0.665L10.675,0.36ZM4.297,3.213C3.908,3.56 3.539,3.93 3.195,4.322L3.296,5.149C3.595,4.773 3.916,4.415 4.256,4.076L4.297,3.213ZM11.874,0.288C11.58,0.288 11.275,0.288 11.005,0.324L10.891,0.599C11.191,0.551 11.49,0.509 11.796,0.48L11.874,0.288ZM12.066,1.001L12.21,0.659C11.822,0.703 11.438,0.763 11.059,0.839L10.867,1.301C11.257,1.187 11.658,1.085 12.066,1.001ZM10.471,1.001C10.16,1.079 9.872,1.169 9.548,1.271L9.279,1.912C9.596,1.768 9.92,1.642 10.25,1.522L10.471,1.001ZM8.631,2.218L8.931,1.481C8.535,1.636 8.146,1.816 7.768,2.008L7.54,2.817C7.894,2.613 8.254,2.41 8.631,2.218ZM5.227,4.651L5.251,3.704C4.795,4.088 4.37,4.507 3.98,4.957L4.028,5.94C4.4,5.486 4.8,5.055 5.227,4.651ZM2.679,4.921C2.324,5.388 2.007,5.883 1.732,6.402L1.978,7.151C2.221,6.683 2.494,6.231 2.793,5.796L2.679,4.921ZM0.347,11.533L0.347,11.922C0.347,12.21 0.347,12.492 0.383,12.773L0.623,13.187L0.623,12.066L0.347,11.533ZM1.63,10.568L1.288,9.848C1.144,10.338 1.029,10.837 0.947,11.341L1.276,12.036C1.37,11.541 1.488,11.051 1.63,10.568ZM0.623,9.327C0.519,9.779 0.443,10.238 0.395,10.699L0.695,11.251C0.749,10.843 0.821,10.442 0.911,10.052L0.623,9.327ZM4.765,3.596C5.087,3.296 5.425,3.014 5.778,2.751L5.898,1.996C5.524,2.229 5.164,2.483 4.819,2.757L4.765,3.596ZM2.421,8.494L2.188,7.553C1.912,8.076 1.675,8.619 1.48,9.177L1.822,9.896C1.994,9.417 2.194,8.949 2.421,8.494Z
```

**Review by adamrusted — CHANGES_REQUESTED** [review]

**Comment by adamrusted** [comment]
Closing due to inactivity.

**Pull request closed without merging**